test/extended/jenkins/plugin.go

```diff
@@ -169,6 +193,80 @@ var _ = g.Describe("[jenkins][Slow] openshift pipeline plugin", func() {
 
 			g.By("get build console logs and see if succeeded")
 			err = waitForJenkinsActivity(fmt.Sprintf("http://%s/job/test-plugin-job/1/console", hostPort), "Finished: SUCCESS", 200)
+			logs, _ := getJenkinsJobLogs(hostPort, "test-plugin-job")
```

Comment:
I think our wires got crossed when we discussed this after scrum.

Rather than inserting the test here, create a new test under `g.Context` with something like 

`g.It("jenkins-plugin test case for env var builds", func() {`

And then another like 

`g.It("jenkins-plugin test case multi tag", func() {`

I do think the `BeforeSuite` change is good.  Bring up Jenkins just once for the (now greater than 1) set of tests